### PR TITLE
Finished problems 51 and 52.

### PR DESCRIPTION
**Pull request by ikhat**

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -4240,6 +4240,22 @@ \section*{Task 2.3: Pat and Jamie}
 			\item $\mat{3\\3\\2}$
 			\item $\mat{1&0&1\\0&1&0\\0&0&1}$.
 		\end{enumerate*}
+		\begin{solution}
+			For each part, we compute the reduced row echelon form of the matrix
+			and simply count the number of pivots.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: omit "simply"

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -4240,6 +4240,22 @@ \section*{Task 2.3: Pat and Jamie}
...
+			\begin{enumerate}
+				\item $\rref\left(\mat{1&1\\2&2}\right)=\mat{1&1\\0&0}$, and so
+					the rank of this matrix is 1.
+				\item $\rref\left(\mat{1&2\\3&4}\right)=\mat{1&0\\0&1}$, and so
+					the rank of this matrix is 2.
+				\item This matrix is already in reduced row echelon form, and so
+					its rank is 2.
+				\item $\rref\left(\mat{3\\3\\2}\right)=\mat{1\\1\\1}$, and so
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: nope

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -4254,13 +4270,75 @@ \section*{Task 2.3: Pat and Jamie}
...
+						example
+						\[
+							\vec b
+							=\mat{1\\1\\-1}
+							=t\mat{1\\1\\-1}+s\mat{2\\2\\-2}
+							=(t+2s)\mat{1\\1\\-1}
+						\]
+						for any real numbers $s$ and $t$ that satisfy $t+2s=1$.
+						In other words, any $\vec x$ of the form $\mat{1-2s\\s\\0}$
+						is a solution to the system, for any $s \in \R$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: The explanation can be dropped. You can just list the systems with no other words. At this point in the course, writing the complete solution should be second nature.

File: problemsets-mat223/linearalbegra.tex (hunk 2 above)
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Whoops. Let's pretend that one never happened.

File: problemsets-mat223/linearalbegra.tex [hunk 4]
```diff
@@ -4240,6 +4240,22 @@ \section*{Task 2.3: Pat and Jamie}
 			\item $\mat{3\\3\\2}$
 			\item $\mat{1&0&1\\0&1&0\\0&0&1}$.
 		\end{enumerate*}
+		\begin{solution}
+			For each part, we compute the reduced row echelon form of the matrix
+			and count the number of pivots.
+			\begin{enumerate}
+				\item $\rref\left(\mat{1&1\\2&2}\right)=\mat{1&1\\0&0}$, and so
+					the rank of this matrix is 1.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Let's move the answer to the beginning of the sentence.

File: problemsets-mat223/linearalbegra.tex [hunk 5]
```diff
@@ -4254,13 +4270,49 @@ \section*{Task 2.3: Pat and Jamie}
 	and the non-augmented matrix of coefficients $A=\mat{1&2&1\\1&2&3\\-1&-2&1}$.
 	\begin{parts}
 		\item What is $\Rank(A)$?
+			\begin{solution}
+				$\rref(A)=\mat{1&2&0\\0&0&1\\0&0&0}$, and so the rank of this
+				matrix is 2.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Same comment

**Pull request merged**